tests/test-cases/Group23-VIC-Machine-Service/23-06-VCH-Certificate.md

```diff
@@ -0,0 +1,17 @@
+Test 23-06 - VCH Certificate
+=======
+
+# Purpose:
+To verify vic-machine-server can provide a VCH host certificate when available
+
+# Test Steps:
+1. Deploy the VCH into the test environment with tls enabled
+2. Verify that the certificate is available using the vic-machine service
+3. Verify that the certificate is available for its particular datacenter using the vic-machine service
+4. Deploy a new VCH into the test environment using --no-tls and --no-tls-verify so that tls is disabled and no cert is created
+5. Verify that the certificate is unavailable (404) using the vic-machine service
+6. Verify that the certificate is unavailable (404) for its particular datacenter using the vic-machine service
```

Comment:
Steps 4-6 don't seem to be implemented? Should they be marked with a TODO for now?

Reply:
I figured it out and just went ahead and implemented them.